app/controllers/redmine_oauth_controller.rb

```diff
@@ -308,6 +310,13 @@ def try_to_login(email, info)
       invalid_credentials
       raise StandardError, l(:notice_account_invalid_credentials)
     end
+
```

Comment:
Remove the extra empty line here.

Reply:
I will, I'm on my vacation right now, though, so sometimes next week.